docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-close-case-proc.adoc

```diff
@@ -1,2 +1,24 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-close-case-proc-{context}']
+= Closing a case
+
+Case instance can be completed when there are no more activities to be performed and the business goal was achieved, or it can be closed prematurely. In the second case, it is often useful to comment why the case instance is being closed. A closed case can be reopened later with the same case ID if required.
+
+Usually the case owner closes the case when all work is completed and the case goals have been met.
+
+Use the following procedure to close a case using the Showcase application.
+
+.Procedure 
+You are logged into the Showcase application and are the owner or administrator for a case instance that you want to close.
```

Comment:
"logged in to", per style guide. Search "log in", "login", and maybe even "into" to be sure. Example, "12.1 Install and login..." should be "...log in".